.travis.yml

```diff
@@ -6,45 +6,43 @@ cache:
 
 env:
   global:
-    - COMPOSER_ARGS="--no-interaction --no-plugins"
+    - COMPOSER_ARGS="--no-interaction"
     - COVERAGE_DEPS="php-coveralls/php-coveralls"
 
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
+    - php: 7.3
       env:
         - DEPS=lowest
-    - php: 5.6
+    - php: 7.3
       env:
         - DEPS=latest
-    - php: 7
-      env:
-        - DEPS=lowest
-    - php: 7
-      env:
-        - DEPS=latest
-    - php: 7.1
+        - BENCHMARKS=true
+        - CS_CHECK=true
+        - TEST_COVERAGE=true
+    - php: 7.4
       env:
         - DEPS=lowest
-    - php: 7.1
+    - php: 7.4
       env:
         - DEPS=latest
-        - BENCHMARKS=true
-        - CS_CHECK=true
-        - TEST_COVERAGE=true
-    - php: 7.2
+    - php: nightly
```

Comment:
Travis support `8.0` now

Reply:
Since when? Tested that 3 days ago and it did not.
Any official resource to verify your statement before I start changing `.travis.yml` twice?

Reply:
https://travis-ci.community/t/php-8-0-missing/10132/23